PULL_REQUEST_TEMPLATE.md

```diff
@@ -37,9 +37,9 @@ Note that if you are using a distribution purpose repository/package, please als
  * [ ] Auto-update filemap is correct.
 
 # Git commit checklist
- * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- * [ ] The parent of the commit(s) in the PR is not old than 3 days.
- * [ ] Pull request is sending from a non-master branch with meaningful name.
+ * [ ] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
+ * [ ] The parent of the commit(s) in the PR is not older than 3 days.
+ * [ ] Pull request is send from a non-master branch with a meaningful name.
```

Comment:
I think maybe it's `sent` not `send`? Actually, I think `sending` could be more precise here?

Reply:
I think I prefer sent, will update.